src/unix/netbsd.c

```diff
@@ -66,22 +66,34 @@ void uv_loadavg(double avg[3]) {
 
 
 int uv_exepath(char* buffer, size_t* size) {
+  /*
+   * Intermediate buffer, retrieving partial path name does not work
+   * As of NetBSD-8(beta), vnode->path translator does not handle files
+   * with longer names than 31 characters
+   */
+  char int_buf[PATH_MAX];
+  size_t int_size = __arraycount(int_buf);
```

Comment:
Can you use `ARRAY_SIZE(int_buf)` here?  (Or `sizeof` since it's a char array.)

How big is PATH_MAX on netbsd?  Could this conceivably overflow the stack?

Reply:
`/usr/include/sys/syslimits.h:#define    PATH_MAX                 1024   /* max bytes in pathname *`

I think we can deal with it.